docs/src/main/asciidoc/building-native-image.adoc

```diff
@@ -322,13 +322,6 @@ Executing [/data/home/gsmet/git/quarkus-quickstarts/getting-started/target/getti
 ...
 ----
 
-[TIP]
-====
-By default, Quarkus waits for 60 seconds for the native image to start before automatically failing the native tests. This
```

Comment:
Why did you remove this section? Shouldn't it have just been updated to the new name?

Reply:
You are right. I'll fix it